### ISSUE TITLE
Hover style on guide "cards" is missing

### ISSUE THREAD

**Issue by willdurand** [mozilla/addons-frontend#7138]
### Describe the problem and steps to reproduce it:

1. Browse the AMO homepage: https://addons-dev.allizom.org/
2. Hover one of the 3 "hero/guide" cards

### What happened?

There is no hover style. Thankfully, the cursor changes but that's not enough.

### What did you expect to happen?

We should have a hover style, similar to the items in the collection shelves as shown below:

![2018-11-30 16 24 58](https://user-images.githubusercontent.com/217628/49297991-8956db80-f4bc-11e8-8eb2-6535a259015f.gif)
